RPi3.md

```diff
@@ -0,0 +1,278 @@
+
+Install UFW:
+```bash
+sudo apt install ufw
+```
+
+This command allows SSH connections from your LAN only. Replace `192.168.1.0` with your own subnet:
+```bash
+sudo ufw allow from 192.168.1.0/24 to any port 22
+```
+
+These ports need to be accessible from anywhere (The default subnet is 'any' unless you specify one):
+```bash
+sudo ufw allow 80
+sudo ufw allow 443
+sudo ufw allow 9735
+```
+
+Verify your configuration:
+```bash
+sudo ufw status
+```
+
+Enable your firewall:
+```bash
+sudo ufw enable
+```
+
+**Step 13** - Reformat flash drive, to be configured as swap space.
+
+**⚠️ Warning:** Using any SD card for swap space **kills it quickly!**. Instead, use a flash drive, as the instructions discuss.
+
+The command `sudo fdisk -l` shows a list of the connected storage devices. Assuming you only have one flash drive connected, it will be
+called `/dev/sda`. Double-check that `/dev/sda` exists, and that its storage capacity matches your flash memory.
+
+Delete existing flash drive partition:
+```bash
+sudo fdisk /dev/sda
+# Press 'd'
+# Press 'w'
+```
+
+Create new primary flash drive partition:
+```bash
+sudo fdisk /dev/sda
+# Press 'n'
+# Press 'p'
+# Press '1'
+# Press 'enter'
+# Press 'enter'
+# Press 'w'
+```
+
+Format partition as ext4:
+```bash
+sudo mkfs.ext4 /dev/sda1
+# Create folder for mount.
+sudo mkdir /mnt/usb
+# Look up UUID of flash drive.
+UUID="$(sudo blkid -s UUID -o value /dev/sda1)"
+# Add mount to fstab.
+echo "UUID=$UUID /mnt/usb ext4 defaults,nofail 0" | sudo tee -a /etc/fstab
+```
+
+Test changes to `fstab` file:
+```bash
+sudo mount -a
+```
+
+Verify that drive is mounted:
+```bash
+df -h
+```
+
+`/dev/sda1` should appear as mounted on `/mnt/usb`.
+
+Create symlink to flash drive for Docker:
+```bash
+sudo mkdir /mnt/usb/docker
+sudo ln -s /mnt/usb/docker /var/lib/docker
+```
+
+**Step 14** - Finally, move Swapfile to USB and increase its size.
+
+Edit its configuration file:
+```bash
+sudo nano /etc/dphys-swapfile
+```
+
+Change the CONF_SWAPFILE line to:
+`CONF_SWAPFILE=/mnt/usb/swapfile`
+
+
+Change the CONF_SWAPSIZE line to:
+`CONF_SWAPSIZE=2048`
+
+Stop and restart the swapfile service:
+```bash
+sudo /etc/init.d/dphys-swapfile stop
+sudo /etc/init.d/dphys-swapfile start
+```
+
+**Step 15** - Install BTCPayServer!
+
+Login as `root`:
+```bash
+sudo su -
+```
+
+Create a folder for BTCPayServer:
+```bash
+mkdir btcpayserver
+cd btcpayserver
+```
+
+Clone the BTCPayServer-Docker repository into the folder:
+```bash
+git clone https://github.com/btcpayserver/btcpayserver-docker
+cd btcpayserver-docker
+```
+
+Set your environment variables. Make sure the `BTCPAY_HOST` value uses your own domain & subdomain. As usual, run each command separately:
+```bash
+export BTCPAY_HOST="btcpay.YourDomain.com"
```

Comment:
WIth a [recent PR](#314) we added some more details on the local deployment (see [this part of the diff](https://github.com/btcpayserver/btcpayserver-doc/pull/328/files#diff-76a73d0660e25e9341734f7560c49f19L233)). This is missing here and in the RPI4 docs – just mentioning it in case we want to integrate it in the new versions as well.

Reply:
ping @normandmickey would you like to tackle this in a different PR?

Reply:
ping @normandmickey you want to tackle this in a separate PR or?